File: exs/mundo_1/c#/025.cs
/*
Desafio 025

Problema: Crie um programa que leia o nome de uma pessoa e diga se ela tem "SILVA" no nome.

Resolução do problema:
*/

using System;

class io {
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    public static void print(object data, string end="\n") {
        string text = data.ToString();

        Console.Write($"{text}{end}");
    }
}

class Program {
  public static void Main () {
    string nome = io.input("Insira um nome: ");

    string[] splitNome = nome.Split(' ');
    bool silva = Array.Exists(splitNome, elem => elem.ToLower() == "silva");

    io.print(silva);
  }
}

File: exs/mundo_1/rust/025.rs
/*
Desafio 025

Problema: Crie um programa que leia o nome de uma pessoa e diga se ela tem "SILVA"
          no nome.
*/

fn main() {
    let mut input = String::new();

    println!("Digite o nome da cidade: ");
    std::io::stdin().read_line(&mut input).unwrap();

    input = input.trim().to_string();

    let tem_silva = input.to_uppercase().contains("SILVA");

    println!("O nome da pessoa digitada tem \"SILVA\": {tem_silva}",);
}
